app/code/Magento/Downloadable/Observer/SaveDownloadableOrderItemObserver.php

```diff
@@ -131,6 +131,13 @@ public function execute(\Magento\Framework\Event\Observer $observer)
                         ScopeInterface::SCOPE_STORE
                     );
                 $linkPurchased->setLinkSectionTitle($linkSectionTitle)->save();
+                
+                $linkstatus =  \Magento\Downloadable\Model\Link\Purchased\Item::LINK_STATUS_PENDING;
```

Comment:
Could you please change the variable name to `$linkStatus`?

Reply:
sure